paasta_tools/cli/schemas/tron_schema.json

```diff
@@ -316,6 +316,10 @@
                     "type": "string",
                     "pattern": "[-a-zA-z0-9_]"
                 },
+                "timeout_spark": {
```

Comment:
Is this used anywhere?
I believe this is to set timeout value for the spark job.
Please verify if we use 'min' or 'm' for parsing minutes in the past. 
If 'min' is used 'min', change help section examples to '30min' instead of '30m'

Reply:
Updated the regex.